modules/virt-hot-plugging-a-virtual-disk-using-the-web-console.adoc

```diff
@@ -0,0 +1,21 @@
+// Module included in the following assemblies:
+//
+// * virtual_machines/virtual_disks/virt-hot-plugging-virtual-disks.adoc
+
+[id="virt-hot-plugging-a-virtual-disk-using-the-web-console{context}"]
+= Hot-plugging a virtual disk using the web console
+
+Hot-plug virtual disks that you want to attach to a virtual machine instance (VMI) while a virtual machine is running.
+
+.Prerequisites
+* You must have a running virtual machine to hot-plug a virtual disk.
+
+.Procedure
+
+. Click *Workloads → Virtualization → Virtual Machines* from the side menu.
```

Comment:
I don't have access to the UI, but based on other web console procedures in our docs, this should be:

. Click *Workloads* → *Virtualization* from the side menu.
. On the *Virtual Machines* tab, select the running virtual machine to which you want to hot-plug a virtual disk. (_you are hot-plugging the disk, so the phrase "add a hot-plugged disk" seems confusing_)
. Click the *Disks* tab.
. On the *Add Disk* page, fill in the information for the virtual disk that you want to hot-plug.
. Click *Add*.